src/index.ts

```diff
@@ -25,15 +55,18 @@ export * from "./api";
  * type.
  */
 export interface TelemetryRecorder<
-  FeatureT extends string,
-  ActionT extends string,
-  MetadataKeyT extends string,
-  BillingProductsT extends string,
-  BillingCategoriesT extends string
+  BillingProducts extends string,
+  BillingCategories extends string
 > {
   /**
-   * Record an event. See each type's documentation for more details now event
-   * attributes.
+   * Record a telemetry event.
+   *
+   * @param feature must be snakeCase and '.'-delimited, e.g. 'myFeature.subFeature'.
```

Comment:
snake case is `like_this`, youre thinking of camel case :smile:

Reply:
doh! will fix 😮‍💨